File: app/src/main/java/at/msd/friehs_bicha/cdcsvparser/general/BaseAppModel.java
package at.msd.friehs_bicha.cdcsvparser.general;

import java.io.Serializable;
import java.util.ArrayList;

import at.msd.friehs_bicha.cdcsvparser.App.AppType;
import at.msd.friehs_bicha.cdcsvparser.App.BaseApp;

public class BaseAppModel implements Serializable {

    public BaseApp txApp;
    public AppType appType;

    public boolean isRunning;

    /**
     * Creates a new AppModel
     *
     * @param appType which app to use
     */
    public BaseAppModel(AppType appType) {
        this.appType = appType;
    }

}

File: app/src/main/java/at/msd/friehs_bicha/cdcsvparser/general/BaseAppModel.kt
package at.msd.friehs_bicha.cdcsvparser.general

import at.msd.friehs_bicha.cdcsvparser.app.AppType
import at.msd.friehs_bicha.cdcsvparser.app.BaseApp
import java.io.Serializable

open class BaseAppModel
/**
 * Creates a new AppModel
 *
 * @param appType which app to use
 */(var appType: AppType) : Serializable {
    var txApp: BaseApp? = null
    var isRunning = false
}